From 00b29f3d19d3ed06cedf4cbec43a365025318025 Mon Sep 17 00:00:00 2001
From: Martchus <martchus@gmx.net>
Date: Sun, 15 Oct 2017 22:25:22 +0200
Subject: [PATCH 2/2] Prevent linking qhelpconverter against static bearer
 plugins

This would lead to linker errors since Qt Network isn't used
by qhelpconverter and hence its dependencies aren't pulled.

FIXME:
* Find out why qmake attempts to link against plugins
  by default which are from a module which isn't even used.
* Fix pulling dependencies of static modules when only
  the plugin, but not the related module itself, is used.
---
 src/assistant/qhelpconverter/qhelpconverter.pro | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/assistant/qhelpconverter/qhelpconverter.pro b/src/assistant/qhelpconverter/qhelpconverter.pro
index d60aae7c..99943618 100644
--- a/src/assistant/qhelpconverter/qhelpconverter.pro
+++ b/src/assistant/qhelpconverter/qhelpconverter.pro
@@ -1,4 +1,5 @@
 QT += help widgets
+QTPLUGIN.bearer=-
 
 SOURCES += conversionwizard.cpp \
            inputpage.cpp \
-- 
2.19.0

